libs/sdk-common/justfile

```diff
@@ -0,0 +1,18 @@
+init:
+    cargo install wasm-pack
+
+[linux]
+wasm-clippy:
+    cargo clippy --target=wasm32-unknown-unknown -- -D warnings -A clippy::uninlined-format-args
+
+[macos]
+wasm-clippy:
+    AR=/usr/local/opt/llvm/bin/llvm-ar CC=/usr/local/opt/llvm/bin/clang cargo clippy --target=wasm32-unknown-unknown -- -D warnings -A clippy::uninlined-format-args
```

Comment:
I think instead of using a justfile we could also do something like this in the `Makefile`


```
UNAME := $(shell uname)

ifeq ($(UNAME), Darwin)
    CLANG_PREFIX += AR=/usr/local/opt/llvm/bin/llvm-ar CC=/usr/local/opt/llvm/bin/clang
endif
```

And then

```
wasm-clippy:
    $(CLANG_PREFIX) cargo clippy --target=wasm32-unknown-unknown -- -D warnings -A clippy::uninlined-format-args
```

Depending on preference to install `just` or not.

Reply:
Done [68dc4a1](https://github.com/breez/breez-sdk-greenlight/pull/1170/commits/68dc4a1b1568ae4906edcc626d4d95ab06752e9b)